api/tests/unit/domain/models/User_test.js

```diff
@@ -132,7 +154,44 @@ describe('Unit | Domain | Models | User', () => {
 
   });
 
-  describe('#email', function() {
+  describe('hasAccessToCertificationCenter', () => {
+
+    it('should be false is user has no access to no CertificationCenter', () => {
```

Comment:
typo `is` => `if` et trop de `no` : `should be false if user has no access to given certification center`

Reply:
ok